framework/util/file_path.cpp

```diff
@@ -435,6 +435,28 @@ void GetApplicationInfo(FileInfo& file_info)
     {
         GFXRECON_LOG_WARNING("Failed to retrieve the application executable name");
     }
+#elif defined(__ANDROID__) || defined(__linux__)
+    std::ifstream cmdlineFile("/proc/self/cmdline");
+
+    if (cmdlineFile.is_open())
+    {
+        std::string line;
+        std::getline(cmdlineFile, line);
+
+        if (!line.empty())
+        {
+            util::platform::StringCopy(file_info.AppName, sizeof(file_info.AppName), line.c_str(), line.size() * sizeof(line.c_str()[0]));
```

Comment:
I would like to get rid of this copy as I noted above, but for later `For [std::string](https://en.cppreference.com/w/cpp/string/basic_string), the elements are bytes` so the multiplication by sizeof is not necessary.